src/main/java/database/service/helper/Settings.java

```diff
@@ -0,0 +1,88 @@
+package database.service.helper;
+
+import code.practice.exceptions.database.CreationDatabaseException;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Properties;
+
+//TODO: logger temporary disabled to do not spam to console, and need to add SLF4J
+//import org.apache.logging.log4j.Logger;
+//import org.apache.logging.log4j.LogManager;
+
+public class Settings {
+    private String propertyFileName;
+    private final Path defaultJsonDatabasePath;
+    static final String DATABASE_STORAGE_PATH = "database.storage.path";
+    static final String DEFAULT_DATABASE_STORAGE_PATH = "database";
+//    private static final Logger logger = LogManager.getLogger(Settings.class);
+
+    public Settings() throws CreationDatabaseException {
+        defaultJsonDatabasePath = getFilePath(DEFAULT_DATABASE_STORAGE_PATH);
+    }
+
+    public Settings(String propertyFileName) throws CreationDatabaseException {
+        this();
+        this.propertyFileName = propertyFileName;
+    }
+
+    public String getPropertyFileName() {
+        return propertyFileName;
+    }
+
+    public void setPropertyFileName(String propertyFileName) {
+        this.propertyFileName = propertyFileName;
+    }
+
+    public Path getDatabasePath() {
+        if (propertyFileName == null) {
+            return defaultJsonDatabasePath;
+        }
+
+        var properties = new Properties();
+        var input = Settings.class.getClassLoader().getResourceAsStream(propertyFileName);
+
+        if (input == null) {
+            return defaultJsonDatabasePath;
+        }
+        try {
+            properties.load(input);
+            String databaseStoragePath = properties.getProperty(DATABASE_STORAGE_PATH);
+            return Path.of(databaseStoragePath);
+        } catch (IOException e) {
+//            logger.error(e);
+            return defaultJsonDatabasePath;
+        }
+    }
+
+    public Path getFilePath(String directoryName) throws CreationDatabaseException {
+        URI resourceFolder;
+        String pathToResourceFolder;
+
+        try {
+            resourceFolder = getClass().getResource(File.separator).toURI();
```

Comment:
`.toURI()` can produce `NullPointerException`, consider catching `Exception` instead of `URISyntaxException` to cover all possible exceptions.